Gruntfile.js

```diff
@@ -7,10 +7,11 @@ module.exports = function(grunt) {
 	grunt.loadNpmTasks('grunt-contrib-clean');
 
 	var DIST = 'dist/',
-		DIST_OPTIMIZED = DIST + 'optimized/',
-		DIST_ANDROID = DIST + 'android/',
+		DIST_OPTIMIZED = DIST + 'svg/optimized/',
 		DIST_IOS = DIST + 'iOS/',
-		DIST_SPRITE = DIST + 'sprite/',
+		DIST_SPRITE = DIST + 'svg/sprite/',
```

Comment:
This will break imports in consumer apps - we could either:

1. bump a major version
2. leave the sprite in `DIST + 'sprite/'`

I favor (2)